lib/NativeCode/DynamicLibraryLoaderHelper/ManagedPluginCode/Application.cs

```diff
@@ -33,7 +33,7 @@ internal static class Application
         /// This is the path of the streaming assets directory relative to the
         /// output directory of this class library.
         /// </summary>
-        public static readonly string streamingAssetsPath = @"..\..\..\..\Assets\StreamingAssets\";
+        public static readonly string streamingAssetsPath = @"..\..\..\..\..\..\Assets\StreamingAssets\";
```

Comment:
Nervousness about relative paths intensify. Why did this change?

Reply:
This changed because the `ManagedPluginCode` project is _only ever_ deployed to `$(SolutionDir)Build\$(Configuration)\$(Platform)` for testing purposes. And the relative path to the `Assets\StreamingAssets\` directory from _that_ directory (which typically evaluates to `[PathToRepo]\lib\NativeCode\DynamicLibraryLoaderHelper\Build\Debug\x64`) is `..\..\..\..\..\..\Assets\StreamingAssets`